LibreNMS/service.py

```diff
@@ -93,7 +93,7 @@ def populate(self):
         self.node_id = os.getenv('NODE_ID')
         self.set_name(config.get('distributed_poller_name', None))
         self.distributed = config.get('distributed_poller', ServiceConfig.distributed)
-        self.group = ServiceConfig.parse_group(config.get('distributed_poller_group', ServiceConfig.group))
+        self.group = ServiceConfig.parse_group(config.get('default_poller_group', ServiceConfig.group))
```

Comment:
This doesn't work, it makes the poller only poll the default group.

Reply:
oops, seems i was to motivated ;-)